### PR TITLE
fix: export admin store config

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

- Exports the admin test store config on an `adminTestUtils` object
- Adjusts releases tests to work with unstable_useDocument

### Why is it needed?

- To make it work 

### How to test it?

- Run the tests or they should pass here in the CI

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/pull/19756

### CODE REVIEW

**Comment by markkaylor** [comment]
@joshuaellis I'm not sure this exactly what you had in mind but gave it my best shot 😄

**Comment by joshuaellis** [comment]
Should we just re-export the whole `tests/utils` thing? that way all the providers are set up too... IMO i'd add a subroute for it though so you import it by doing `@strapi/admin/strapi-admin/tests` then it's isolated. 

I think benefit of that, MSW can be added to jest automatically if you want it using all the endpoints we use etc. We could just write something on the server that says it's experimental and never stable so if we change and endpoint, people can't get annoyed but the monorepo can share data and hooks too.

lmk what you think 👍🏼

**Comment by markkaylor** [comment]
> Should we just re-export the whole `tests/utils` thing? that way all the providers are set up too... IMO i'd add a subroute for it though so you import it by doing `@strapi/admin/strapi-admin/tests` then it's isolated.
> 
> I think benefit of that, MSW can be added to jest automatically if you want it using all the endpoints we use etc. We could just write something on the server that says it's experimental and never stable so if we change and endpoint, people can't get annoyed but the monorepo can share data and hooks too.
> 
> lmk what you think 👍🏼

Yeah I think that makes sense...the only thing I'm a little unsure of is: won't plugins needs to extend the Providers store? For example content-releases needs to add its middleware etc... My initial thought is I can set a defaultStore that is exported as well from the admin `tests/utils` and add a store prop that is passed form render to the Providers to override the default, or export Providers as well so plugins can still compose there own render, or both.... 

Is that what you had in mind or do you have another idea?

**Comment by markkaylor** [comment]
Good idea that looks clean and consistent with the other options, I'll give it a try!

**Comment by joshuaellis** [comment]
> I still to need figure this bit out:
> 
> > import it by doing @strapi/admin/strapi-admin/tests
> 
> but I think it should build with the tests passing now 🤞

let me know if you need any help!

File: packages/core/content-releases/admin/tests/server.ts
Comment (joshuaellis) [inline packages/core/content-releases/admin/tests/server.ts:1]: do you still need this? 🤔 

We export `server` from the admin test utils now – should you use that server for your tests? I think you can adhoc still add endpoints?

**Pull request merged**